src/newsletter-editor/styling/index.js

```diff
@@ -178,7 +204,7 @@ export const Styling = compose( [
 				id={ `inspector-custom-css-control-${ instanceId }` }
 				label={ __( 'Custom CSS', 'newspack-newsletters' ) }
 				help={ __(
-					'Custom CSS will be appended to default styles in sent emails only. Refresh editor to preview styles.',
+					'This is an advanced feature and may result in unpredictable behavior. Custom CSS will be appended to default styles in sent emails only.',
```

Comment:
This means that the styles will not be applied to the public newsletters viewed as posts? That would actually be a nice feature, WDYT?

Reply:
Correct—it was my hunch that the CSS entered for emails may not translate well to the site. In my experience, email CSS requires a lot of `!important` and aggressive style overrides, which could be too heavy-handed for the site, so I felt that if publishers wanted to apply their custom styles to public newsletters viewed as posts, they could do that via the Customizer and have a bit more control. I'm willing to try, though, maybe in a new PR?

Reply:
> if publishers wanted to apply their custom styles to public newsletters viewed as posts, they could do that via the Customizer and have a bit more control. 

That makes sense, yeah.